man/rgbasm.5

```diff
@@ -41,7 +41,6 @@ or
 Labels tie a name to a specific location within a section (see
 .Sx Labels
 below).
-They must come first in the line.
```

Comment:
This doc already said "Each line may have components in this order" above, and said that label definitions and macro invocations must come "at the beginning of a line" in their relevant sections; so calling out labels specifically here is just confusing (e.g. directives are first in their line too, why aren't they mentioned?).

Reply:
I think it's an unclear vestigial statement from the time where labels had to be column-zero. It can be safely removed.